docs/source/predictors/latest.rst

```diff
@@ -13,9 +13,8 @@ algorithm but rather approximates the latest software stack resolution.
 The randomness introduced causes incompatibilities with pip/Pipenv/Poetry
 resolver.
 
-Check `termial random
-<https://medium.com/@fridex/termial-random-for-prioritized-picking-an-item-from-a-list-a65a4f563224>`__
-and `its optimized form
-<https://medium.com/@fridex/optimizing-termial-random-by-removing-binomial-coefficient-e39b9ca7aaa3>`__
-which is used to prioritize picking more recent releases considering version
-specifier.
+..  Check `termial random
+  <https://medium.com/@fridex/termial-random-for-prioritized-picking-an-item-from-a-list-a65a4f563224>`__
+  and `its optimized form
+  <https://medium.com/@fridex/optimizing-termial-random-by-removing-binomial-coefficient-e39b9ca7aaa3>`__
+  which is used to prioritize picking more recent releases considering version specifier.
```

Comment:
I need to re-publish this one. It was moved to a more open platform but sits there as a draft. Feel free to remove it.

Reply:
Commented it out for now. Can update this link as well once you publish it.